website/docs/gator.md

```diff
@@ -264,6 +266,25 @@ the `run` flag:
 gator verify path/to/suites/... --run "disallowed"
 ```
 
+### Validating Generated Resources with ExpansionTemplates
```

Comment:
Not sure if this is too much or if its not enough (i.e. no example or a more complete example).